src/include/function/cast/cast_functions.h

```diff
@@ -34,7 +34,7 @@ struct CastToString {
     template<typename T>
     static inline std::string castToStringWithVector(
```

Comment:
I would remove `WithVector` from the naming. You can always use same name with overloading.